src/change_stream.ts

```diff
@@ -576,8 +581,14 @@ function createChangeStreamCursor<TSchema>(
   changeStream: ChangeStream<TSchema>,
   options: ChangeStreamOptions
 ): ChangeStreamCursor<TSchema> {
-  const changeStreamStageOptions: Document = { fullDocument: options.fullDocument || 'default' };
-  applyKnownOptions(changeStreamStageOptions, options, CHANGE_STREAM_OPTIONS);
+  const changeStreamStageOptions: Document = applyKnownOptions(
```

Comment:
Up to you, but maybe this helper isn't worth having I'm thinking that the array we're passing in here is exactly 4 keys it might be worth it to just hardcode the work since it's reletively small.